Buform.Example.MvvmCross.iOS/DigitGenerationCell.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using Buform.Example.Core;
+using CoreGraphics;
+using Foundation;
+using SByteDev.Common.Extensions;
+using UIKit;
+
+namespace Buform.Example.MvvmCross.iOS
+{
+    [Preserve(AllMembers = true)]
+    [Register(nameof(DigitGenerationCell))]
+    public sealed class DigitGenerationCell : FormCell<DigitGenerationItem>
+    {
+        private const string GoForwardIconName = "goforward";
+
+        private UILabel? _titleLabel;
+        private UILabel? _valueLabel;
+        private UIButton? _button;
+
+        public DigitGenerationCell()
+        {
+            /* Required constructor */
+        }
+
+        public DigitGenerationCell(IntPtr handle) : base(handle)
+        {
+            /* Required constructor */
+        }
+
+        private void UpdateReadOnlyState()
+        {
+            if (Item == null)
+            {
+                return;
+            }
+
+            if (_button == null)
+            {
+                return;
+            }
+
+            _button.Enabled = !Item.IsReadOnly;
+        }
+
+        private void UpdateLabel()
+        {
+            if (_titleLabel == null)
+            {
+                return;
+            }
+
+            _titleLabel.Text = Item?.Label;
+        }
+
+        private void UpdateValue()
+        {
+            if (_valueLabel == null)
+            {
+                return;
+            }
+
+            _valueLabel.Text = Item?.Value.ToString();
+        }
+
+        private void ExecuteCommand(object _, EventArgs __)
+        {
+            Item?.RegenerateCommand.SafeExecute();
+        }
+
+        protected override void Initialize()
+        {
+            SelectionStyle = UITableViewCellSelectionStyle.None;
+            ContentView.UserInteractionEnabled = true;
+
+            _titleLabel = new UILabel
+            {
+                TranslatesAutoresizingMaskIntoConstraints = false,
+                Font = UIFont.PreferredBody,
+                TextColor = UIColor.Label
+            };
+
+            _valueLabel = new UILabel
+            {
+                TranslatesAutoresizingMaskIntoConstraints = false,
+                Font = UIFont.PreferredBody,
+                TextColor = UIColor.SecondaryLabel
+            };
+
+            _button = UIButton.FromType(UIButtonType.System);
+            _button.Frame = new CGRect(0, 0, 70, 40);
+            _button.TouchUpInside += ExecuteCommand;
```

Comment:
The handler method must be called like the handler itself, but with `On` prefix to clearly indicate the connected event.

Reply:
`button.TouchUpInside += OnButtonTouched` -> `button.TouchUpInside += OnTouchUpInside`